cryptofeed/feed.py

```diff
@@ -47,6 +47,8 @@ def __init__(self, address: Union[dict, str], timeout=120, timeout_interval=30,
         book_interval: int
             Number of updates between snapshots. Only applicable when book deltas are enabled.
             Book deltas are enabled by subscribing to the book delta callback.
+        candle_interval: str
+            Length of time between a candle's Open and Close.
```

Comment:
you might want to mention that this is only a valid kwarg on exchanges that support candles

Reply:
Yeah, to be honest, this comment should be at the exchange level but it seems there's no docstring inheritance. Child classes overwrite the parent's docstring, huh?